src/redux/books/books.js

```diff
@@ -1,29 +1,86 @@
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const GET_CURRENT_API = 'bookStore/books/GET_CURRENT_API';
+const GET_CURRENT_API_SUCCESS = 'bookStore/books/GET_CURRENT_API_SUCCESS';
+const GET_CURRENT_API_FAILURE = 'bookStore/books/GET_CURRENT_API_FAILURE';
+const API = `
+  https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/0ePu6tv5E4TBgiHo9yos/books`;
 
-const initialState = [];
+const initialState = {
+  status: 'Idle',
+  books: {},
+};
+
+export const getApi = async (dispatch) => {
+  dispatch({ type: GET_CURRENT_API });
+  const books = await fetch(API).then((res) => res.json());
+  return dispatch({ type: GET_CURRENT_API_SUCCESS, books });
+};
 
-export const addBook = (payload) => ({
-  type: ADD_BOOK,
-  payload,
-});
+export const removeBook = (payload) => async (dispatch) => {
+  const response = await fetch(API + payload, {
+    method: 'DELETE',
+    mode: 'cors',
+    headers: {
+      'Content-Type': 'application/json',
+      Accept: 'application/json',
+    },
+  }).then((res) => res.status);
+  const result = response === 201 ? 'Book removed' : 'Fail to remove';
+  dispatch(getApi);
+  return dispatch({ type: REMOVE_BOOK, result });
+};
```

Comment:
- [ ] As requested of you by the previous reviewer,  Kindly note that it is a requirement that you make use of [createasyncthunk](https://redux-toolkit.js.org/api/createAsyncThunk) when making your API calls according to the project requirements.